blueoil/cmd/train.py

```diff
@@ -194,6 +194,7 @@ def start_training(config, profile_step):
         max_steps = int(train_dataset.num_per_epoch / config.BATCH_SIZE * config.MAX_EPOCHS)
     else:
         max_steps = config.MAX_STEPS
+    max_steps = max(1, max_steps)
```

Comment:
If you just want to avoid the error from Progbar, what about exiting from this function? Or do you want to run always at least 1 step?

Reply:
Thank you for your suggestion.
I think it's better to show error and exit.

My first idea is sometimes it's hard to know that this error caused by `no. of the train_data*config.MAX_EPOCHS < config.BATCH_SIZE `, so when this happens I correct the max_steps for them to 1.